README.md

```diff
@@ -177,8 +177,28 @@ nix profile install 'github:martinvonz/jj'
 ```
 
 
+#### Homebrew
+
+If you use linuxbrew, you can run:
+
+```shell script
+brew install jj
+```
+
+
 ### Mac
 
+#### Homebrew
+
+If you use linuxbrew, you can run:
```

Comment:
I suppose it shouldn't be "linuxbrew" here (since this is under "Mac")?

Reply:
Done!